src/components/search/FacetsModal.tsx

```diff
@@ -33,17 +34,9 @@ const FacetsModal = (props: FacetsModalProps) => {
       <div className="h-px bg-brand-gray-300 w-screen -mx-4 mb-4 sm:mx-0 sm:w-full">
         {/* Divider */}
       </div>
-      <StandardFacets
-        collapsible={false}
-        showOptionCounts={true}
-        customCssClasses={{
-          standardFacetsContainer: "w-full overflow-y-auto",
-          titleLabel: "text-lg font-bold text-brand-primary",
-          optionLabel: "text-sm",
-          divider: "w-full bg-white my-3",
-        }}
-        searchOnChange={true}
-      />
+      <div className="overflow-y-scroll">
```

Comment:
It's better to use `auto` instead of `scroll` because `scroll` will still show the scrollbar even if there's no overflow.
```suggestion
      <div className="overflow-y-auto">
```